### PR TITLE
chore: format integration tests

### PR DESCRIPTION
**Pull request by f01dab1e**
# Description

```bash
find . -type f -name Nargo.toml -exec sh -c 'cd "$(dirname "{}")" && nargo fmt &> /dev/null || echo "Failed in: $(dirname "{}")"' \;
```
```
Failed in: ./tooling/nargo_cli/tests/test_libraries/bad_name
Failed in: ./tooling/nargo_cli/tests/compile_failure/depend_on_bin
Failed in: ./tooling/nargo_cli/tests/compile_failure/package_name_empty
Failed in: ./tooling/nargo_cli/tests/compile_failure/custom_entry_not_found
Failed in: ./tooling/nargo_cli/tests/compile_failure/workspace_missing_toml
Failed in: ./tooling/nargo_cli/tests/compile_failure/workspace_missing_toml/crates/b
Failed in: ./tooling/nargo_cli/tests/compile_failure/invalid_dependency_name
Failed in: ./tooling/nargo_cli/tests/compile_failure/package_name_hyphen
```

## Documentation\*

Check one:
- [x] No documentation needed.
- [ ] Documentation included in this PR.
- [ ] **[Exceptional Case]** Documentation to be submitted in a separate PR.

# PR Checklist

- [x] I have tested the changes locally.
- [x] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

File: noir_stdlib/src/sha512.nr [hunk 1]
```diff
@@ -10,69 +9,135 @@ fn rotr64(a: u64, b: u64) -> u64 // 64-bit right rotation
...
-    
+fn sha_c(msg: [u64; 16], hash: [u64; 8]) -> [u64; 8] {
+    let K: [u64; 80] = [4794697086780616226,
+        8158064640168781261,
```
Comment (f01dab1e) [inline noir_stdlib/src/sha512.nr:55]: Oh, that's a rather unpleasant formatting

File: noir_stdlib/src/hash/poseidon/bn254/consts.nr
Comment (f01dab1e) [inline noir_stdlib/src/hash/poseidon/bn254/consts.nr:1]: oh, what is it...

File: noir_stdlib/src/hash/poseidon/bn254.nr [hunk 2]
```diff
@@ -73,180 +65,176 @@ fn absorb<M,N,O,P>(
     rate: Field, // Rate
     capacity: Field, // Capacity; usually 1
     msg: [Field; P] // Arbitrary length message
-) -> [Field; O] {
-
+) -> [Field; O]
+{
```
Comment (f01dab1e) [inline noir_stdlib/src/hash/poseidon/bn254.nr:69]: we really should address this issue

**Comment by f01dab1e** [comment]
@jfecher

**Review by jfecher — APPROVED** [review]
Lets merge this now. There are some remaining parts that may be a bit questionable but overall I think there is a lot more good. For the remaining bits we can open issues for them and tackle them at a later date.

**Comment by TomAFrench** [comment]
I'm really not a fan of how the formatter is collapsing code sections separated with a blank line together if there's a comment between them. This severely hampers readability for me.

**Comment by f01dab1e** [comment]
@TomAFrench, ok, I'll fix it

**Pull request merged**